src/plugins/discover/public/application/context/context_app_content.test.tsx

```diff
@@ -15,12 +15,16 @@ import { SortDirection } from '@kbn/data-plugin/public';
 import { ContextAppContent, ContextAppContentProps } from './context_app_content';
 import { LoadingStatus } from './services/context_query_state';
 import { dataViewMock } from '../../__mocks__/data_view';
-import { DiscoverGrid } from '../../components/discover_grid/discover_grid';
 import { discoverServiceMock } from '../../__mocks__/services';
 import { DocTableWrapper } from '../../components/doc_table/doc_table_wrapper';
-import { KibanaContextProvider } from '@kbn/kibana-react-plugin/public';
+import { ServicesContextProvider } from '../services_provider';
 import { buildDataTableRecord } from '../../utils/build_data_record';
 
+const MockDiscoverGrid = () => <></>;
+jest.mock('../../components/discover_grid/discover_grid', () => ({
+  DiscoverGrid: jest.fn(() => <MockDiscoverGrid />),
+}));
```

Comment:
No, the test was passing, but complaining about an update: `Warning: An update to DiscoverGrid inside a test was not wrapped in act(...).`.
I can add the async `act` after the `mountWithIntl` if you want, but it would meach changing the whole test to async. Since here we were only checking the _DiscoverGrid_ was rendered inside I just mocked the component, it should make no difference.

Reply:
If it's not much of a trouble, would be great to update tests and keep the original `DiscoverGrid`. Thanks for updating other tests!